instance/config.py

```diff
@@ -0,0 +1,48 @@
+#instance/config.py
+"""
+    This module holds api configuration.
+"""
+import os
+basedir = os.path.abspath(os.path.dirname(__file__))
+
+
+class Config(object):
+    """
+        Parent configuration class.
+    """
+    DEBUG = False
+    CSRF_ENABLED = True
+    SECRET = os.getenv('SECRET')
+    TESTING = False
+    JWT_ALGORITHM = 'HS256'
+    JWT_SECRET_KEY = 'jtw-very-supa-secret'
```

Comment:
you can also use the os.random function to create better secret keys

Reply:
Sure, let me see how I can implement that.

Reply:
Let me see how I can implement that.